src/styles/index.css

```diff
@@ -154,4 +152,58 @@ input::placeholder {
 
 button {
   border: none;
+  background-color: white;
+  color: #4386bf;
+}
+
+.progress {
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #307bbe;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+  margin-right: 15px;
+}
+
+.azure-btn {
+  background-color: #0290ff !important;
+  color: #e8e8e8 !important;
+}
+
+.progress_div {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.chapter {
+  margin-bottom: 2rem;
+  align-items: flex-start;
+}
+
+@media screen and (max-width: 1199px) {
+  .book_list {
+    text-align: center;
+    margin-bottom: 2rem;
+  }
+
+  .book_list ul {
+    justify-content: center;
+  }
+
+  .progress_div {
+    margin-bottom: 2rem;
+    font-family: 'Montserrat', sans-serif;
+  }
+
+  .chapter {
+    margin-bottom: 2rem;
+    align-items: center;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .list-item {
+    margin: 2.313rem 1.25rem 0;
+  }
```

Comment:
There are some differences in the design in terms of the positioning and alignment of the elements in each book components as shown below :point_down: 

The elements in each book component such as the book title and author, the progress icon,  the chapter information, and the update progress bar are supposed to be aligned horizontally but yours is aligned vertically. Kindly check and fix this anomaly to make your project match the actual design. 

### Your design 

![Screenshot from 2023-03-04 13-22-26](https://user-images.githubusercontent.com/65233860/222904172-0f28c873-4a75-42ac-9e65-b91310265a89.png)


### The actual design 

![Screenshot from 2023-03-04 13-07-07](https://user-images.githubusercontent.com/65233860/222904184-ab1cc6c1-2bb4-484e-8e35-df669dc5dd34.png)

Reply:
Hello @jerryowusu this happens on screens width equal to or smaller than 1199px. I added this for responsiveness as you can see on line 184. On a larger screen the design is the same.

Reply:
I just reduced the break point to 991px I hope its ok